packages/web-components/src/components/masthead/left-nav.ts

```diff
@@ -256,15 +270,6 @@ class C4DLeftNav extends StableSelectorMixin(CDSSideNav) {
           ${c4dPrefix}-masthead-composite`
         )
         ?.querySelector(`${c4dPrefix}-masthead`);
-      if (expanded && !this._importedSideNav) {
-        import('./left-nav-name');
-        import('./left-nav-menu');
-        import('./left-nav-menu-section');
-        import('./left-nav-menu-item');
-        import('./left-nav-menu-category-heading');
-        import('./left-nav-overlay');
-        this._importedSideNav = true;
-      }
```

Comment:
`super.updated()`, called at the top of this function, is where we try to set focus. We have to make sure all our components are loaded before that, so this section is moved to `willUpdate()` to prioritize loading in the components we need.